### DISCUSSION TITLE
Error PytzUsageWarning

### DISCUSSION THREAD

**Discussion by 5uru** [kevinzg/facebook-scraper#633] [Q&A] [answered]
hi when i run the code i get the following error: /home/nuru_ai/anaconda3/envs/nuru/lib/python3.9/site-packages/dateparser/date_parser.py:35: PytzUsageWarning: The localize method is no longer necessary, as this time zone supports the fold attribute (PEP 495). For more details on migrating to a PEP 495-compliant implementation, see https://pytz-deprecation-shim.readthedocs.io/en/latest/migration.html
  date_obj = stz.localize(date_obj)

**Comment by neon-ninja** [comment, marked as answer]
Duplicate of https://github.com/kevinzg/facebook-scraper/issues/601
